usbtmc/usbtmc.py

```diff
@@ -696,51 +718,77 @@ def clear(self):
             raise UsbtmcException("Clear failed", 'clear')
 
     def remote(self):
-        "Send remote command"
+        """
+        Send the remote command to the instrument.
+        :return:
+        """
         raise NotImplementedError()
 
     def local(self):
-        "Send local command"
+        """
+        Send the local command to the instrument.
+        :return:
+        """
         raise NotImplementedError()
 
     def lock(self):
-        "Send lock command"
-
+        """
+        Advantest/ADCMT ONLY: Send the lock command to the instrument.
```

Comment:
For all these Advantest/ADCMT only methods, it would be nicer for the base `Instrument` class to not have these methods on it. Rather, there should be an `AdvantestInstrument` class that implements those, deriving from `Instrument`.